### ISSUE TITLE
Suggestion: Give users the option to hide/unhide mods

### ISSUE THREAD

**Issue by Eiion** [Prospector/ModMenu#376]
I'd like to suggest to add a feature that allows users to manually hide/unhide mods from the Mod Menu.

Just like we can make Mods visible that are hidden, it would be nice if we were able hide certain mods ourselves. Say you run the game with certain "main mods" which you're always using and that are set to our liking, hiding them would help with keeping a better overview of new/other/testing/... mods to only see them.

It could be as simple as offering clickable check boxes to mark the mods which we want to hide and at the next time mod menu gets opened, they are hidden. To see them again we can simply switch to show hidden mods (including those automatically hidden) to make them visible again and thus to remove the checkmark if we want to permanently unhide them again.

Or it could be more sophisticated by adding a whole new listing of the installed mods that allows us to mark/un-mark those mods we'd like to hide/unhide. But I don't think that would be necessary - the most simple integration of such a feature would be enough (for me at least).

Thanks for considering.

**Comment by Madis0** [comment]
The feature already exists in the config file, do you think the UI is necessary for it?

**Comment by Eiion** [comment]
Yes, most definitely. Being able to do it via the UI is a massive quality of live improvement compared to having to fiddle with config files.
